Resources/views/change_password/index.html.twig

```diff
@@ -0,0 +1,39 @@
+{% block title %}{% endblock %}
+
+{% set hide_navigation_second = true %}
+{% set hide_page_header = true %}
+{% set page_class = 'login-container' %}
+
+{% block body %}
+    <div>
+
+        {% set formErrors = form.vars.errors.form.getErrors(true) %}
+
+        <div id="form_errors"
```

Comment:
je kan ook het if-statement om de form_errors heen zetten, je hebt hem nu uit de admin bundle geplukt en daar is hij zo omdat er JS validatie op zit en het element dan al in de DOM moet zitten maar dat is hier niet het geval.

En kijk even in de twig file van de 2fa/login welke styling ik heb toegepast voor een alert box.